### DISCUSSION TITLE
node kode/Kha --server returns an error

### DISCUSSION THREAD

**Discussion by Fhalo48** [Kode/Kha#1402] [General] [answered]
Hi

When testing HTML5 on local Windows 10 desktop. 
Ran node kode/Kha --server, but, it returns an error:-

f:\KodeStudio\resources\app\kodeExtensions\kha\Kha\Tools\khamake\khamake.js:1
(function (exports, require, module, __filename, __dirname) { let versions = p
                                                                  ^^^^^^^^

Any suggestions?

Thanks

**Comment by RobDangerous** [comment, marked as answer]
Same answer as in #1400, Kode Studio is not up to date - and even when it is up to date again, please use a recent version of Kha when you run it from a command-line, the integrated copy in Kode Studio is really just there to get started.
